### PR TITLE
Feed filters fe

### PR DESCRIPTION
**Pull request by camhashemi**
@andrewconner

a huge thank you to @cvializ and @JRuff42 for all the help 

screen shot 2016-01-08 at 6 08 30 pm
screen shot 2016-01-08 at 6 11 02 pm

### CODE REVIEW

**Comment by cvializ** [comment]
I'll have to see it to give feedback, let me know when is good for us to go over it

File: kifi-backend/modules/shoebox/angular/src/feed/feed.tpl.html [hunk 1]
```diff
@@ -27,12 +34,10 @@
...
   </section>
 
-  <div class="kf-feed-header-container" ng-if="!(isLoading() && feed.length == 0)">
-    <div kf-card-style-selector></div>
-  </div>
-
-  <section class="kf-feed-item" ng-if="::((!isLoading() || undefined) && feed.length < 10)">
-    <div class="kf-feed-top"><h3>Add a keep</h3></div>
+  <span></span>
+  <section class="kf-feed-add" ng-if="(!isLoading() || undefined) && feed.length < 10">
```
Comment (cvializ) [inline kifi-backend/modules/shoebox/angular/src/feed/feed.tpl.html]: what are the empty `<span></span>`s accomplishing?

`(!isLoading() || undefined)` can probably be just `!isLoading()`. (not that you originally wrote it, just a thought I have now)

**Comment by cvializ** [comment]
it's looking killer! :hocho:

File: kifi-backend/modules/shoebox/angular/src/feed/feed.tpl.html (hunk 1 above)
Comment (camhashemi) [inline kifi-backend/modules/shoebox/angular/src/feed/feed.tpl.html]: yeah I have no idea about the `|| undefined`, will remove. Empty spans are going to be cleaned up.

**Comment by cvializ** [comment]
:tada: can't wait to see it live :+1:

**Comment by camhashemi** [comment]
releasing behind the admin exp for QA

**Pull request merged**